.gitignore

```diff
@@ -0,0 +1 @@
+*~
```

Comment:
remove this change. this is system related. I do not use editors that create that backup files. This should be in your `~/.git/info/exclude`